tools/common/FileCopier.cs

```diff
@@ -149,15 +194,31 @@ static CopyFileResult CopyFileCallback (CopyFileWhat what, CopyFileStep stage, I
 			}
 		}
 
+		public static bool IsUptodate (string source, string target, ReportErrorCallback reportErrorCallback, LogCallback logCallback, bool check_contents = false, bool check_stamp = true)
```

Comment:
```suggestion
		public static bool IsUpToDate (string source, string target, ReportErrorCallback reportErrorCallback, LogCallback logCallback, bool check_contents = false, bool check_stamp = true)
```

Reply:
I think maybe this one and the ones below should have this casing, but I do see that there are already existing overloads with that casing so maybe not

Reply:
Yeah, I wondered about this too, but decided to not change it to not make the PR bigger than it needs to be.

Reply:
If you apple that, make sure all instances are done too. I have made the mistake of using git commit suggestion to have later a build error :/